layouts/community/numpad_5x4/mrsendyyk/readme.md

```diff
@@ -0,0 +1,19 @@
+# Sendy YK's Numpad 5x4 Layout and Keymap
+
+[https://mr.sendyyk.com](https://mr.sendyyk.com)
+
+## Numpad 5x4 Layout
+
+![LAYOUT_numpad_5x4](https://raw.githubusercontent.com/mrsendyyk/my_qmk/master/soyuz/assets/layout-numpad-5x4.png)
```

Comment:
Note: These numbers are meant to describe the matrix wiring for the board being used with the keymap. These numbers are accurate for the Soyuz, but some boards with this layout wire things differently.

Reply:
Yes